dbt/adapters/databricks/impl.py

```diff
@@ -139,6 +146,47 @@ def list_relations_without_caching(  # type: ignore[override]
             )
         ]
 
+    def _list_relations_with_information(
+        self, schema_relation: DatabricksRelation
+    ) -> List[Tuple[DatabricksRelation, str]]:
+        kwargs = {"schema_relation": schema_relation}
+        try:
+            # The catalog for `show table extended` needs to match the current catalog.
+            with self._catalog(schema_relation.database):
+                results = self.execute_macro(SHOW_TABLE_EXTENDED_MACRO_NAME, kwargs=kwargs)
+        except dbt.exceptions.RuntimeException as e:
+            errmsg = getattr(e, "msg", "")
+            if (
+                "[SCHEMA_NOT_FOUND]" in errmsg
```

Comment:
If there is an error code in the error message, will we get the error message details as well ("Database ... not found")?

Reply:
The error message was changed:

> [SCHEMA_NOT_FOUND] The schema \<schemaName> cannot be found. Verify the spelling and correctness of the schema and catalog.
If you did not qualify the name with a catalog, verify the current_schema() output, or qualify the name with the correct catalog.
To tolerate the error on drop use DROP SCHEMA IF EXISTS.